helloworld/BUILD

```diff
@@ -12,6 +12,16 @@ python_binary(
     ]
 )
 
+# Note: Has sdist dependencies, so must be built on Linux.
+python_awslambda(
+    name='helloworld-awslambda',
+    sources = ["awslambda.py"],
+    dependencies=[
+        "helloworld/greet",
+    ],
+    handler='helloworld.awslambda:handler',
+    runtime='python3.7',
```

Comment:
We might want to update the V1 deprecation in Pants to mention that this must be specified. I don't think we need to, because the error message for required fields should be good, but it is nicer to proactively explain the new field.